packages/atlas/src/components/_auth/LogInModal/LogInModal.tsx

```diff
@@ -0,0 +1,137 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import { Text } from '@/components/Text'
+import { Tooltip } from '@/components/Tooltip'
+import {
+  Container,
+  StyledAppLogo,
+  StyledButton,
+  StyledHideSvg,
+  StyledShowSvg,
+  TextCointainer,
+} from '@/components/_auth/LogInModal/LogInModal.styles'
+import { SignInModalStepTemplate } from '@/components/_auth/SignInModal/SignInSteps/SignInModalStepTemplate'
+import { Button } from '@/components/_buttons/Button'
+import { FormField } from '@/components/_inputs/FormField'
+import { Input } from '@/components/_inputs/Input'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { atlasConfig } from '@/config'
+import { LogInErrors, useLogIn } from '@/hooks/useLogIn'
+import { useSnackbar } from '@/providers/snackbars'
+
+export const LogInModal = () => {
+  const [isLoading, setIsLoading] = useState(false)
+  const [isPasswordShown, setPasswordShown] = useState(false)
+  const handleLogIn = useLogIn()
+  const { displaySnackbar } = useSnackbar()
+  const {
+    register,
+    handleSubmit,
+    setError,
+    formState: { errors },
+  } = useForm<{ email: string; password: string }>({
+    resolver: (data, ctx, options) => {
+      const resolver = zodResolver(
+        z.object({
+          email: z.string().email(),
+          password: z.string().min(1, 'Please provide password'),
+        })
+      )
+
+      return resolver(data, ctx, options)
+    },
```

Comment:
```suggestion
    resolver: zodResolver(
      z.object({
        email: z.string().email(),
        password: z.string().min(1, 'Please provide password'),
      })
    ),
```

this should be the same